config/config-builder/src/validator_config.rs

```diff
@@ -218,9 +227,19 @@ impl ValidatorConfig {
                 .test
                 .as_ref()
                 .and_then(|config| config.publishing_option.clone()),
-        ));
+        );
+
+        let waypoint = if self.build_waypoint {
+            Some(db_bootstrapper::compute_genesis_waypoint::<LibraVM>(
+                genesis.clone(),
+            )?)
```

Comment:
I suggest extending this on two lines for clarity

```
let waypoint = db_bootstrapper::compute_genesis_waypoint::<LibraVM>(genesis.clone())?;
Some(waypoint);
```

Reply:
why though?

Reply:
I find it clearer to read : o